source/payment_flow/index.html.md.erb

```diff
@@ -13,25 +13,13 @@ There's a different process for [taking Direct Debit payments](/direct_debit/#ta
 
 ## Overview
 
-The following diagram outlines the payment process for a service that has a GOV.UK Pay
-integration:
+The following payment process is for a service that is integrated with the GOV.UK Pay API.
```

Comment:
at some point it might be good to say - there is a different process for payment links and provide a link with details on that? recognise it's a dicsussion as to whether that sits in the product pages or the tech docs.

Reply:
Directly above this on the page at https://docs.payments.service.gov.uk/payment_flow/#how-gov-uk-pay-works is a warning with a link to the payment links page in the tech docs: `Warning - This does not apply to your users who use the payment links functionality`. Is that sufficient do you think? I don't want to repeat the same link again so soon.

Reply:
yeah that's fine, sorry hadnt seen that. would be good to think about payment links specific content at some point too.